src/components/CytoscapeLayout/__tests__/CytoscapeLayout.test.tsx

```diff
@@ -11,11 +11,14 @@ jest.mock('../../../services/Api');
 jest.useFakeTimers();
 
 const testNamespace = 'ISTIO_SYSTEM';
+const testHandler = () => {
```

Comment:
What will be the goal of this handler? :)

Reply:
This is a test, so I don't understand well...

Reply:
@israel-hdez you can simulate a mouse click in the test.   `testHandler()` function allows you to assert that on-click logic gets call (as it would in the real code)